routes/dao_coin_exchange.go

```diff
@@ -688,9 +689,9 @@ func CalculateScaledExchangeRateFromPriceString(
 	// Beyond this point, we know that the operation type is lib.DAOCoinLimitOrderOperationTypeBID
 
 	// Scale up the price to account for DAO Coin -> DESO trades
-	if IsDesoPkid(buyingCoinPublicKeyBase58Check) {
```

Comment:
I think there was some other reason DH wanted this change. I think it matches both DESO and the zero public key